features/networking/ipv6_dualstack.feature

```diff
@@ -0,0 +1,78 @@
+Feature: ipv6 dual stack cluster test scenarios
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-40581
+  @admin
+  Scenario: Perject should be isolation when using multitenant policy for ipv6 dual stack
+    # create project and pods
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    Given I obtain test data file "networking/list_for_pods_ipv6.json"
+    When I run the :create client command with:
+      | f | list_for_pods_ipv6.json |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(0).ip_v4` is stored in the :p1pod1ipv4 clipboard
+    And evaluation of `pod(0).ip_v6` is stored in the :p1pod1ipv6 clipboard
+    Given I use the "test-service" service
+    And evaluation of `service.ip_v4` is stored in the :service_ipv4 clipboard
+    And evaluation of `service.ip_v6` is stored in the :service_ipv6 clipboard
+
+    # create another project and pods
+    Given I create a new project
+    And evaluation of `project.name` is stored in the :proj2 clipboard
+    Given I obtain test data file "networking/list_for_pods_ipv6.json"
+    When I run the :create client command with:
+      | f | list_for_pods_ipv6.json |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(1).name` is stored in the :p2pod1 clipboard
+
+    When I execute on the "<%= cb.p2pod1 %>" pod:
+      | curl | --connect-timeout | 5 | <%= cb.p1pod1ipv4 %>:8080 |
+    Then the step should succeed
+    And the output should contain "Hello"
+    When I execute on the "<%= cb.p2pod1 %>" pod:
+      | curl | --connect-timeout | 5 | [<%= cb.p1pod1ipv6 %>]:8080 |
+    Then the step should succeed
+    And the output should contain "Hello"
+
+    #access pod by service ipv4 and ipv6 address
+    When I execute on the "<%= cb.p2pod1 %>" pod:
+      | curl | --connect-timeout | 5 | <%= cb.service_ipv4 %>:27017 |
+    Then the step should succeed
+    And the output should contain "Hello"
+
+    When I execute on the "<%= cb.p2pod1 %>" pod:
+      | curl | --connect-timeout | 5 | [<%= cb.service_ipv6 %>]:27017 |
```

Comment:
so we have to remember to use `[]` for IPv6 when used as a URL, unless we create an `ipv6_url` method.  Or maybe a general `to_url` method.